src/webgpu/api/validation/state/device_lost/destroy.spec.ts

```diff
@@ -454,6 +454,142 @@ Tests creating render pipeline on destroyed device.
     }, awaitLost);
   });
 
+g.test('createComputePipelineAsync')
```

Comment:
WDYT about merging the variants with a `awaitLost` param?

Reply:
Merged.

I also ended up turning `awaitLost` into a param not a subcase param. I realized that as a subcase param, it was reusing the device between subcases! which meant that the lost device was already lost before .destroy()'ed in some subcases

Reply:
That is - overall, anything that destroys the device shouldn't use subcases, I think.

Reply:
Oh great catch!